probes/codeReviewTwoReviewers/impl.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//nolint:stylecheck
+package codeReviewTwoReviewers
+
+import (
+	"embed"
+	"fmt"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/clients"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/utils"
+)
+
+var (
+	//go:embed *.yml
+	fs               embed.FS
+	reviewerLoginErr = fmt.Errorf("could not find the login of a reviewer")
+)
+
+const (
+	probe            = "codeReviewTwoReviewers"
+	minimumReviewers = 2
+)
+
+func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
+	rawReviewData := &raw.CodeReviewResults
+	return codeReviewRun(rawReviewData, fs, probe, finding.OutcomePositive, finding.OutcomeNegative)
+}
+
+// Looks through the data and validates author and reviewers of a changeset
+// Scorecard currently only supports GitHub revisions and generates a positive
+// score in the case of other platforms. This probe is created to ensure that
+// there are a number of unique reviewers for each changeset.
+
+func codeReviewRun(reviewData *checker.CodeReviewData, fs embed.FS, probeID string,
+	positiveOutcome, negativeOutcome finding.Outcome,
+) ([]finding.Finding, string, error) {
+	changesets := reviewData.DefaultBranchChangesets
+	var findings []finding.Finding
+	foundHumanActivity := false
+	leastFoundReviewers := 0
+	nChangesets := len(changesets)
+	if nChangesets == 0 {
+		return nil, probeID, utils.NoChangesetsErr
+	}
+	// Loops through all changesets, if an author login cannot be retrieved: returns OutcomeNotAvailabe.
+	// leastFoundReviewers will be the lowest number of unique reviewers found among the changesets.
+	for i := range changesets {
+		data := &changesets[i]
+		if data.Author.Login == "" {
+			f, err := finding.NewNotAvailable(fs, probeID, "Could not retrieve the author of a changeset.", nil)
+			if err != nil {
+				return nil, probeID, fmt.Errorf("create finding: %w", err)
+			}
+			findings = append(findings, *f)
+			return findings, probeID, nil
+		} else if !data.Author.IsBot {
```

Comment:
Would be cleaner if we did the same thing as in the other probe L63-65, `if` instead of `else if`

Reply:
done in the new file probes/codeReviewOneReviewers/impl.go